tests/suites/firewall/task.sh

```diff
@@ -8,16 +8,21 @@ test_firewall() {
 
 	setup_awscli_credential
 
+	setup_gcloudcli_credential
+
 	echo "==> Checking for dependencies"
-	check_dependencies juju aws
+	check_dependencies juju aws gcloud
```

Comment:
Can we please have another switch statement on the provider to only install these if we are actually testing ec2 or gce.